api/routes/api.js

```diff
@@ -13,27 +13,27 @@ const GetUsersDevicesSchema = {
 	required: ['user_id'],
 	properties: {
 		user_id: {
-			type: 'string'
+			type: 'string' 
 		}
 	}
 };
 
 const GetDeviceDataSchema = {
 	type: 'object',
-	required: ['device_id'],
+	required: ['device_id', 'data_run'],
 	properties: {
 		device_id: {
 			type: 'string'
 		},
 		data_run: {
-			type: 'string'
+			type: 'number'
 		}
 	}
 };
 
 const PostDeviceDataSchema = {
```

Comment:
Do we want to remove the data_run entirely from the post request?

Reply:
I don't think so - it is helpful to have the data_run included in each document in the database, I think it will make sorting by data_run for display and visualization easier to keep it in the post request.

Reply:
Aren't you just replacing it when you insert the data into the database though?

Reply:
Yes - when/where are these schemas checked?

Reply:
PostDeviceDataSchema is checked in api/routes/device I think. When the device posts a datapoint.

Reply:
You're right, I just checked. I will remove data_run from the post request because I set the data_run in the create() function from deviceData.js which is called after the request is validated with the PostDeviceDataSchema.